npm-scripts.js

```diff
@@ -163,6 +161,13 @@ switch (task)
 		break;
 	}
 
+	case 'install-clang-tools':
```

Comment:
Why is this needed at all? Nobody calls `node npm-scripts.js install-clang-tools`.

Reply:
ok, yes, package.json does. What I mean is: does it make sense to call something from.... forget it, it's ok.